package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "nocode",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "scripts": {
+    "test": ""
+  },
+  "repository": {
+    "type": "",
+    "url": ""
+  },
+  "keywords": [],
+  "author": "",
+  "license": ""
+}
```

Comment:
It feels like there's too much code in this file, it doesn't quite follow the philosophy of the project. 
Consider something like this instead:
```json
{}
```

Reply:
Good suggestion! Thanks!

Reply:
Id remove those two brackets, they look ugly.

Reply:
how about: ` `

Reply:
@andrewmclagan are you sure it's backwards compatible with Node v0.0.0?